src/docmanager/index.ts

```diff
@@ -399,7 +442,10 @@ class DocumentManager {
   }
 
   private _data: { [key: string]: Private.IDocumentData } = Object.create(null);
-  private _fileTypes: { [key: string ]: string[] } = Object.create(null);
+  private _modelFactories: { [key: string]: IModelFactory } = Object.create(null);
```

Comment:
I'll answer. Because IE11 only supports iterating through a map using `forEach`. Objects are much more flexible that way, especially with typescript compiling down things like `for (i of object)`, etc.